scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -130,8 +130,23 @@ sealed trait SCollection[T] extends PCollectionWrapper[T] {
    * [[SCollection]].
    */
   def applyTransform[U: ClassTag](transform: PTransform[_ >: PCollection[T], PCollection[U]])
```

Comment:
I wonder if we should make this a `TypeTag` and get `K` and `V` types directly. But then we might as well fix replace `ClassTag` with `TypeTag` everywhere?

Reply:
Gave it a try but am going to abandon this idea.
`applyTransform` is used in a lot of places and it'd require type signature change everywhere.